src/lib/components/RoutingPerformance.svelte

```diff
@@ -0,0 +1,96 @@
+<script lang="ts">
+  import { convertValue } from '$lib/conversion'
+  import { BitcoinDenomination } from '$lib/types'
+  import { formatValueForDisplay } from '$lib/utils'
+  import { channelsAPY$, settings$ } from '$lib/streams'
+  import { onMount } from 'svelte'
+  import type { ChannelAPY } from '$lib/backends'
+  import { translate } from '$lib/i18n/translations'
+
+  let net: ChannelAPY | undefined
+
+  onMount(() => {
+    net = channelsAPY$.value.data?.filter((item) => item.account === 'net')[0]
+  })
```

Comment:
```suggestion
$: net = channelsAPY$.value.data?.filter((item) => item.account === 'net')[0]
```

Can make this trigger based on data and then it will update if the data gets updated.